plugin/trino-hive/src/main/java/io/trino/plugin/hive/TrinoViewHiveMetastore.java

```diff
@@ -173,4 +173,19 @@ public Optional<ConnectorViewDefinition> getView(SchemaTableName viewName)
                         view.getParameters(),
                         view.getOwner()));
     }
+
+    public void setViewComment(ConnectorSession session, SchemaTableName viewName, Optional<String> comment)
```

Comment:
This method is used only from Delta Lake connector, but it exists in Hive connector. Is it possible to extract `io.trino.plugin.iceberg.catalog.hms.TrinoHiveCatalog#updateViewComment` into here and reuse between Delta Lake and Iceberg?